RELEASE.rst

```diff
@@ -1,3 +1,31 @@
+Release 0.3.1
+==============
+
+Major Features and Improvements
```

Comment:
I guess this empy section is deliberate. Personally, I'd remove it (the release number should suggest nothing major has happened), or write "None" in the description.